docs/io/uws.rst

```diff
@@ -1,10 +1,417 @@
-*********************
-UWS (``pyvo.io.uws``)
-*********************
+******************************************
+Universal Worker Service (``pyvo.io.uws``)
+******************************************
+
+.. currentmodule:: pyvo.io.uws
+
+Introduction
+============
+
+The Universal Worker Service (UWS) is an IVOA Recommendation that defines a protocol for managing asynchronous jobs in IVOA services
+through a RESTful API that can be used to submit, monitor and control
+asynchronous operations.
+
+Getting Started
+===============
+
+UWS job documents can be parsed directly from URLs, files, or strings using the
+parsing functions provided by this module:
+
+.. doctest-skip::
```

Comment:
Can you make sure to remove the skips here and below once this is ready for review? Thanks!

Reply:
What is the suggested way to enable these doctests that demonstrate access to UWS async endpoints? Is there a recommended way to mock these, or do you rather point them to actual working services (In which case I'd need to adjust the content based on what the service we choose returns?)

Reply:
I would prefer it if they were pointing to a service. That would also provide an extra test which doesn't hurt in general

Reply:
Ok I've changed the examples to point to an actual service. I do wonder though if this is a slightly fragile approach for the doc tests, since they now depend on an external service, and the build may break if either the service is down, or perhaps that table being queried is no longer available.